src/utils/viewer/viewer.cpp

```diff
@@ -156,8 +163,9 @@ Key Commands:
   'q': Query the agent's state and print to terminal.
 
   Utilities:
+  'c' switch ortho/perspective camera.
```

Comment:
FYI, in [magnum-player](https://magnum.graphics/showcase/player/) I'm using `Num5` to switch between the two, which matches what Blender uses.

Reply:
I'll do that too then.